docs/api/en/math/Frustum.html

```diff
@@ -96,7 +96,7 @@ <h3>[method:this set]( [param:Plane p0], [param:Plane p1], [param:Plane p2], [pa
 		<h3>[method:this setFromProjectionMatrix]( [param:Matrix4 matrix] )</h3>
 		<p>
 		[page:Matrix4 matrix] - Projection [page:Matrix4] used to set the [page:.planes planes]<br /><br />
-		Sets the frustum planes from the projection matrix.
+		Sets the frustum planes from the projection matrix. The plane order will be: right, left, bottom, top, far, near.
```

Comment:
I'm not sure about exposing any type of plane order information in context of `THREE.Frustum`. Even if the order is defined in the description of `setFromProjectionMatrix()`, devs could assume the defined order is always true. However, this is not necessarily the case. The actual plane order in `THREE.Frustum` should not matter.

@WestLangley What are your thoughts about this?

Reply:
I am inclined to agree with @Mugen87. It begs the question: "Why does the order matter?". And of course, it doesn't -- in typical usage.

Reply:
I needed to specifically test against the camera's near plane. In what scenario would `planes[5]` not be the near plane (other than maybe passing invalid arguments to `Matrix4.makePerspective()`?

Reply:
The constructor, `.set()`, and `.copy()` can result in planes in an arbitrary order.

Reply:
Specifically I made the note in the section of `Frustum.setFromProjectionMatrix()` where the plane order is determined mathematically. See: 

https://github.com/mrdoob/three.js/blob/8d961db00cd2367074c33787dd95baf5c2886dcf/src/math/Frustum.js#L54-L59

The note is irrelevant when setting the frustum any other way (`.set()`, `.copy()`, etc.) because as you said anyone can set the planes in any order.

Reply:
Instead of modifying the docs how about updating the code like so:
```js
 planes[ 0 ].setComponents( me3 - me0, me7 - me4, me11 - me8, me15 - me12 ).normalize(); // right
 planes[ 1 ].setComponents( me3 + me0, me7 + me4, me11 + me8, me15 + me12 ).normalize(); // left
 planes[ 2 ].setComponents( me3 + me1, me7 + me5, me11 + me9, me15 + me13 ).normalize(); // bottom
 planes[ 3 ].setComponents( me3 - me1, me7 - me5, me11 - me9, me15 - me13 ).normalize(); // top
 planes[ 4 ].setComponents( me3 - me2, me7 - me6, me11 - me10, me15 - me14 ).normalize(); // far
 planes[ 5 ].setComponents( me3 + me2, me7 + me6, me11 + me10, me15 + me14 ).normalize(); // near
```

Reply:
I would say the information is a lot more visible and easy to find in the docs. Is there an advantage to inlining it with the source code? If the intention is to hide it because it's not always necessarily true, it's probably best to either mention that in the docs too or cancel the PR altogether.

Reply:
Are the inline comments correct for both orthographic camera and perspective camera?

Are they also correct for `WebGPURenderer`, which assumes a different coordinate system handedness?

Reply:
@luisfonsivevo Thanks for the proposed PR. It resulted in #24382, so all was not lost. :-)